website/src/data/users.js

```diff
@@ -31,6 +31,15 @@
 
 // prettier-ignore
 const users = [
+  {
+    title: 'AttoBot',
+    description: 'A multi-purpose Discord bot with many features and API integrations that will enhance your Discord experience.',
+    preview: require('./showcase/attobot.png'),
+    website: 'https://attobot.xyz',
+    source: 'https://github.com/attobot-discord/website',
+    fbOpenSource: false,
+    pinned: true,
+  },
```

Comment:
can you please use the alphabetical order?

Reply:
yes

Reply:
wait

Reply:
done